Source/CoreClasses/Game.swift

```diff
@@ -13,30 +13,41 @@ public class Game: NSObject {
   @objc public var appID: String
   @objc public var name: String
   @objc public var price: Double = 0.00
+  @objc public var shortDescription: String
+  @objc public var developers:Array<String> = []
+  @objc public var publishers:Array<String> = []
 
-  @objc public init(gameName name: String, gamePrice price: Double) {
+  @objc public init(gameName name: String, gamePrice price: Double, shortDescription: String = "") {
 
     self.name = name
     self.price = price
     appID = ""
+    self.shortDescription = shortDescription
     
-    super.init()
-
   }
 
-  @objc public init(appid: String, name: String) {
+  @objc public init(appid: String, name: String, shortDescription: String = "") {
 
     appID = "\(appid)"
     self.name = name
+    self.price = 0.00
+    self.shortDescription = shortDescription
 
   }
 
   public func toString() -> String {
     return name
   }
-
-  public func reduceMemoryLoad() {
-
+  
+  public static func defaultValue() -> Game {
```

Comment:
Wouldn't you rather have this as a computed variable ?
Would be closer to the way you use it

```
public static var defaultValue: Game {
    let game = Game(appid: "N/A", name: "No Game")
    game.price = 0.00
    game.developers.append("No Developers")
    game.publishers.append("No Publishers")

    return game
}
```

Reply:
True true, I have done this. However, git is not allowing pushes to this branch